### PR TITLE
New modal

### PR DESCRIPTION
**Pull request by MayconDionisio**

### CODE REVIEW

**Comment by raphamorim** [comment]
Hi @MayconDionisio, can you adjust lint tests?

**Comment by MayconDionisio** [comment]
Sorry for that @raphamorim, i will adjust the lint tests and other things.

File: dist/modal.css.js [hunk 1]
```diff
@@ -1,67 +1,66 @@
...
-    color: "#333",
-    position: "relative",
-    fontFamily: '"Open Sans", sans-serif',
-    fontSize: "16px"
-  },
-  hidden: {
-    visibility: "hidden"
+    color: '#333',
+    position: 'relative',
+    fontFamily: '"Open Sans" "sans-serif"',
```
Comment (ricobl) [inline dist/modal.css.js]: The comma is missing here...

**Review by ricobl — CHANGES_REQUESTED** [review]
Apart from the font, looks good to me! Thanks!

File: dist/modal.css.js (hunk 1 above)
Comment (raphamorim) [reply dist/modal.css.js]: Check this refactor: `fontFamily: ['Open Sans', 'sans-serif'],`

Comment (MayconDionisio) [reply dist/modal.css.js]: Ok, i'll do that. Thanks!

**Pull request merged**